ddtrace/contrib/rq/__init__.py

```diff
@@ -0,0 +1,236 @@
+"""
+The RQ integration will trace your jobs.
+
+Worker Usage
+~~~~~~~~~~~~
+
+``ddtrace-run`` can be used to easily trace your workers::
+
+    DD_WORKER_SERVICE_NAME=myworker ddtrace-run rq worker
+
+
+See https://github.com/DataDog/trace-examples/tree/kyle-verhoog/rq/python/rq for a working example.
+
+
+Configuration
+~~~~~~~~~~~~~
+
+.. py:data:: ddtrace.config.rq['distributed_tracing_enabled']
+
+   If ``True`` the integration will connect the traces sent between the queuer and the RQ worker.
+
+   Default: ``False``
+
+.. py:data:: ddtrace.config.rq['service_name'] # Or via DD_RQ_SERVICE_NAME environment variable
```

Comment:
Any ideas for how we can better represent the ways and precedence for configuration methods? Maybe a table?

Reply:
We should already expose a table with all those config variables :(

Reply:
I believe we do for tracer-level environment variables but not for integration specific variables.